auacm/app/templates/index.html

```diff
@@ -90,5 +90,6 @@
   <script src="/static/js/controllers/SettingsController.js"></script>
   <script src="/static/js/controllers/ViewProblemController.js"></script>
   <script src="/static/js/controllers/EditProblemController.js"></script>
+  <script src="/static/js/controllers/EditCompetitionController.js"></script>
```

Comment:
I think they are combined. I just realized, however, that I forgot to rename the controller in the `app.controller(...` definition.